modules/lvms-upgrading-lvms-on-sno.adoc

```diff
@@ -6,17 +6,59 @@
 [id="lvms-upgrading-lvms-on-sno_{context}"]
 = Upgrading {lvms} on {sno} clusters
 
-Currently, it is not possible to upgrade from {rh-storage} Logical Volume Manager Operator 4.11 to {lvms} 4.12 on {sno} clusters.
+You can upgrade the {lvms-first} Operator to align with your {sno} version.
 
-[IMPORTANT]
-====
-The data will not be preserved during this process.
-====
+.Prerequisites
+
+* You have upgraded your {sno} cluster.
+
+* You have installed a previous version of the {lvms} Operator.
+
+* You have installed the OpenShift CLI (`oc`).
+
+* You have logged in as a user with `cluster-admin` privileges.
```

Comment:
You could simplify this language a bit by making bullets 1-3 preterite. 

Is the final item really the first step of the procedure? If so, you might sub that with something like  "You have a user account that has `cluster-admin` privileges" and instruct the reader to log in as procedure step 1.

Reply:
Leaving as is for clarity and because this format used widely in the docs